projects/BFDR.CLI/Program.cs

```diff
@@ -12,11 +26,19 @@ class Program
 @"* BFDR Command Line Interface - commands
   - help:  prints this help message  (no arguments)
   - fakerecord: prints a fake JSON birth record (no arguments)
+  - description: prints a verbose JSON description of the record (in the format used to drive Canary) (1 argument: the path to the birth record)
+  - 2ije: Read in the FHIR XML or JSON birth record and print out as IJE (1 argument: path to birth record in JSON or XML format)
+  - 2ijecontent: Read in the FHIR XML or JSON birth record and dump content  in key/value IJE format (1 argument: path to birth record in JSON or XML format)
+  - ije2json: creates an IJE birth record and prints out as JSON
+  - ije2json: Read in the IJE birth record and print out as JSON (1 argument: path to birth record in IJE format)
```

Comment:
one of the ije2json commands appears to take multiple files, update the description to say this takes a list of ije files to translate